trackingC.Rmd

```diff
@@ -330,3 +350,35 @@ ggplot(relimp_out, aes(year, value, fill = param)) +
   labs(x = "Year",
        y = "Fraction of relative importance")
 ```
+# Comparing Hector temperature data to CMIP6
+``` {r cmip}
+# Read in comparison data
+cmip <- read.csv("CMIP6_annual_tas_global.csv") %>%
+  # filter by experiment, stop x axis at 2300
+  filter(variable == "Tgav",
+         experiment == "ssp245") %>%
+  group_by(year) %>%
+  summarize(min = min(value),
```

Comment:
This works but since `min` and `max` are the names of R functions, maybe change your column names here for clarity?